neuralmonkey/learning_utils.py

```diff
@@ -536,7 +552,8 @@ def _print_examples(dataset: Dataset,
                     outputs: Dict[str, List[Any]],
                     val_preview_input_series: Optional[List[str]] = None,
                     val_preview_output_series: Optional[List[str]] = None,
-                    num_examples=15) -> None:
+                    num_examples=15,
+                    print_result_to_file="") -> None:
```

Comment:
Since the argument should contain the file name directly, I would call it `result_file` or `output_file`. Also having default value `None` is more pythonic than the empty string.